src/server/scripts/World/go_scripts.cpp

```diff
@@ -266,6 +267,31 @@ class go_resonite_cask : public GameObjectScript
     }
 };
 
+/*######
+## go_proudtusk_remains
+######*/
+
+enum ProudtuskRemains
+{
+    NPC_SHANIPROUDTASK = 9136,
+    QUEST_DREADMAULROCK = 3821
+};
+
+class go_proudtusk_remains : public GameObjectScript
+{
+public:
+    go_proudtusk_remains() : GameObjectScript("go_proudtusk_remains") { }
+
+    bool OnGossipHello(Player* player, GameObject* go) override
+    {
+        // xinef: prevent spawning hundreds of them
```

Comment:
Remove xinef comment, not necessary :P

Reply:
I thought its a bickname of the guy whos code i used to fix this. But if u say.